src/gui/transferlistwidget.cpp

```diff
@@ -761,6 +771,14 @@ void TransferListWidget::applyToSelectedTorrents(const std::function<void (BitTo
     }
 }
 
+void TransferListWidget::removeTorrents(const QVector<BitTorrent::TorrentHandle *> &torrents, const bool isDeleteFileSelected) const
```

Comment:
I supposed you access "selectedTorrents" inside it...
Your variant seems can be free function in anonymous namespace.

Reply:
moved.